tests/ci/azureml_tests/test_groups.py

```diff
@@ -146,8 +147,7 @@
         #
         "tests/smoke/examples/test_notebooks_pyspark.py::test_als_pyspark_smoke",  # 49.53s
         "tests/integration/examples/test_notebooks_pyspark.py::test_als_pyspark_integration",  # 110.58s
```

Comment:
Have other spark tests been passed with AzureML while the benchmark one failed? interesting....

Reply:
yeah, honestly, maybe it spark is an early evil AGI or something lol